doc/specs/vulkan/chapters/memory.txt

```diff
@@ -813,9 +810,9 @@ include::../api/structs/VkMemoryAllocateInfo.txt[]
   * pname:sType is the type of this structure.
   * pname:pNext is `NULL` or a pointer to an extension-specific structure.
   * pname:allocationSize is the size of the allocation in bytes
-  * pname:memoryTypeIndex is the memory type index, which selects the
-    properties of the memory to be allocated, as well as the heap the memory
-    will come from.
+  * pname:memoryTypeIndex is an index selecting a memory type from the
```

Comment:
nit: suggest "identifying" rather than "selecting" - that's what we use elsewhere in the spec.

Reply:
Do you? Can't find anything exept two occurences, which are struct output from the Implementation though.

Reply:
Probably mean "determines"?

Reply:
No I definitely meant identify. Again we don't always use it, and it's not always in the context of an index, but it's in the spec. Search for "identif" and you should find a number of uses.